python/cudf/cudf/io/parquet.py

```
@@ -614,6 +615,30 @@ def _read_parquet(
     # Simple helper function to dispatch between
     # cudf and pyarrow to read parquet data
     if engine == "cudf":
+        # Temporary error to probe a parquet file
+        # and raise decimal128 support error.
+        if len(filepaths_or_buffers) > 0:
+            try:
+                metadata = pq.read_metadata(filepaths_or_buffers[0])
+            except TypeError:
```

What is this branch handling? I always find it helpful to see comments in except clauses ignoring an error, it's very helpful in debugging subtle bugs down the line if the caught exception is too broad.

---

Added a comment to explain the same.